File: src/main/scala/com/equalinformation/tutorial/scala/P1_CommandLineArgsApp.scala
package com.equalinformation.tutorial.scala

/**
 * Created by bpupadhyaya on 9/3/15.
 *
 * P1 : Program to demonstrate simple command line arguments passing - some
 * programming practices will be refined later
 */
object P1_CommandLineArgsApp extends App {
  var loopCount = 0;
  while (loopCount < args.length) {
    println(args(loopCount))
    loopCount += 1
  }
}

File: src/main/java/com/equalinformation/tutorial/java/P1_CommandLineArgsApp.java
package com.equalinformation.tutorial.java;

/**
 * Created by bpupadhyaya on 9/3/15.
 */
public class P1_CommandLineArgsApp {
    public static void main(String...args) {
        int loopCount = 0;
        while(loopCount < args.length) {
            System.out.println(args[loopCount]);
            loopCount += 1;
        }
    }
}
